### ISSUE TITLE
add tool to create and download the DEM derived data

### ISSUE THREAD

**Issue by iona5** [initze/thaw-slump-segmentation#159]
As ArcticDEM derived data (relative elevation, slope) is required for running, we should provide a tool to download those. 

Tasks:
* Define workflow for end user to select spatial extend (RapidEye Tile IDs, bounding box, arbitrary vector data ??) of data to download
* think about a tiling scheme - if we use RapidEye Tiles, we need to provide that data as well in some way
* use already existing processing chain for download (get_ArcticDEM_slope(), get_ArcticDEM_elevation() )
* create CLI tool and/or Notebook (parameters: spatial extend, download folder)

current implementation: https://github.com/initze/thaw_slump_preprocessing/blob/main/DEM_01_ProcessArcticDEM.ipynb

**Comment by initze** [comment]
good point, I have a notebook for that, will add the link shortly to formalize this a bit better!
